packages/node/src/behavior/state/managed/Datasource.ts

```diff
@@ -223,11 +226,28 @@ interface CommitChanges {
 function configure(options: Datasource.Options): Internals {
     const values = new options.type() as Val.Struct;
 
+    let storedValues = options.store?.initialValues;
+
+    let featuresKey: undefined | string;
+    if (options.supervisor.featureMap.children.length) {
+        featuresKey = [...options.supervisor.supportedFeatures].join(",");
+        if (storedValues?.[FEATURES_KEY] !== undefined && storedValues[FEATURES_KEY] !== featuresKey) {
+            logger.warn(
+                `Ignoring persisted values for ${options.path} because features changed from "${values.featuresKey}" to "${featuresKey}"`,
```

Comment:
should this log not also use "storedValues?.[FEATURES_KEY]" ?!

Reply:
yes, fixed